exercises/04-deploy-to-cf.md

```diff
@@ -38,17 +38,22 @@ This is what the service instances should look like in the SAP Cloud Platform co
 ![SCP Services](images/scp-services.png)
 
 ## Adjust the manifest.yml file
-Let's adjust the [manifest.yml](manifest.yml), the configuration file for your Cloud Foundry application.
+Let's adjust the [manifest.yml](../manifest.yml), the configuration file for your Cloud Foundry application.
 Take a look at the services section, where we reference the previously created services instances.
-Replace all occurences of `<participantId>` in the manifest.yml file.
+Replace all occurrences of `<participantId>` in the manifest.yml file.
 
 ## Build and push your application
 Run the following to build and package your application:
 ```sh
 npm run ci-build && npm run ci-package
 ```
 
-Then push it to Cloud Foundry:
+Then, open the [package.json](../package.json), replace the `<participantId>` in the script `cds-deploy:hana` and deploy the data defined in the CSVs to the HDI container:
+```sh
+npm run cds-deploy:hana
```

Comment:
Minor: It might be slightly less confusing to just add this command to the manifest.yml (run `npm run cds-deploy:hana && node dist/index.js`) and just mention what this does instead of having everyone doing this manually.

Reply:
good point, unfortunately, it will result in the error below:
```
2020-04-09T14:14:14.980+0000 [APP/PROC/WEB/0] OUT [cds.deploy] - Starting deploy to SAP HANA ...
2020-04-09T14:14:14.981+0000 [APP/PROC/WEB/0] OUT [cds.deploy] - Creating build tasks
2020-04-09T14:14:14.981+0000 [APP/PROC/WEB/0] OUT [cds.deploy] - Running build
2020-04-09T14:14:16.880+0000 [APP/PROC/WEB/0] OUT [cds.deploy] - Running 'cf marketplace -s hana' with options {}
2020-04-09T14:14:16.888+0000 [APP/PROC/WEB/0] ERR [ERROR] Command 'cf' not found. Make sure you have the Cloud Foundry command line tool installed.
2020-04-09T14:14:16.888+0000 [APP/PROC/WEB/0] ERR     at Function._getError (/home/vcap/deps/0/node_modules/@sap/cds/bin/deploy/to-hana/cfUtil.js:149:20)
2020-04-09T14:14:16.888+0000 [APP/PROC/WEB/0] ERR     at Function.hasMarketplaceService (/home/vcap/deps/0/node_modules/@sap/cds/bin/deploy/to-hana/cfUtil.js:22:24)
2020-04-09T14:14:16.888+0000 [APP/PROC/WEB/0] ERR     at processTicksAndRejections (internal/process/task_queues.js:97:5)
2020-04-09T14:14:17.002+0000 [APP/PROC/WEB/0] OUT Exit status 1
```
The `cf` command is then needed for the cds-deploy:hana.
I guess this is not designed for running on the cloud.

Reply:
For this question, we can open another issue to clarify.

Reply:
Thanks for checking that. This was rather an idea than a necessity. It is fine as is 😊